codebuild_specs/scripts/post_publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+REGISTRY="https://registry.npmjs.org"
+PACKAGE_NAME="amplify-category-api"
+NEW_TAG="latest"
+RETRIES=30
+DELAY=60 # 1 minute delay between retries
```

Comment:
This waits for overall 30 mins for the released version to be available on NPM. With my testing with a tag release, it succeeded within 2 retires.